packages/http-server/src/__tests__/body-params-validation.spec.ts

```diff
@@ -247,25 +248,24 @@ describe('body params validation', () => {
           security: [],
         },
       ]);
+      address = await server.listen(Math.ceil(30000 + Math.random() * 30000), '127.0.0.1');
```

Comment:
body-params and server.oas can run in parallel leading to ADDRINUSE errors. Before, we did not need to worry about this because of `fastify.inject`. I guess I can make the port the same across single test suite but not across different ones. I also saw somewhere a lib for dynamic assignment of ports (more clever than the Math.random solution).

Btw. how single port can help use identify the tests not-closing-handles?